packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -47,75 +44,108 @@
   }
 
   .heading {
-    @apply text-n1-wrap
-    text-color-2
-    pointer-events-none
-    break-words
-    font-medium
-    duration-150
-    ease-in-out;
+    color: var(--calcite-color-text-2);
+    font-size: var(--calcite-font-size--1);
+    font-weight: 500;
+    line-height: 1.20313rem;
+    overflow-wrap: break-word;
+    pointer-events: none;
   }
   .large-visual {
-    @apply items-center
-      text-center;
-    min-block-size: theme("spacing.48");
-    .icon {
-      @apply flex
-        justify-center
-        self-end;
+    align-items: center;
+    min-block-size: 12rem;
+    text-align: center;
+    calcite-icon {
+      align-self: self-end;
+      block-size: 64px;
+      inline-size: 64px;
+      justify-self: center;
     }
     .content-container {
-      @apply self-center;
+      align-self: center;
     }
   }
   .description {
-    @apply text-n2-wrap
-    text-color-3
-    break-words
-    pointer-events-none
-    duration-150
-    ease-in-out;
+    color: var(--calcite-color-text-3);
+    font-size: var(--calcite-font-size--2);
+    font-weight: 400;
+    line-height: 1.03125rem;
+    overflow-wrap: break-word;
+    pointer-events: none;
   }
 }
+
+:host([scale="s"]) {
+  .container {
+    gap: var(--calcite-spacing-sm);
+  }
+  .heading {
+    font-size: var(--calcite-font-size--2);
+    line-height: 1.03125rem;
+  }
+  .description {
+    font-size: var(--calcite-font-size--3);
+    line-height: 0.85938rem;
+  }
+}
+
+:host([scale="l"]) {
+  .container {
+    gap: var(--calcite-spacing-lg);
+  }
+  .heading {
+    font-size: var(--calcite-font-size-0);
+    line-height: 1.375rem;
```

Comment:
Is there a design token or TW util we could use for line height?

Reply:
I'll let @SkyeSeitz and @alisonailea chime in on this.  Ali informed me that there aren't Figma variables for certain token types, including `line-height`. 
![image](https://github.com/Esri/calcite-design-system/assets/821864/d7b140dc-385d-4015-ad10-7ae52c6a3111)

I am noticing some line-height css variables in calcite-design-tokens, but only 1 of the values on the Tile Scale Figma file matches up with any of these, which is `1.375rem` for `--calcite-font-line-height-relative-snug`.

![image](https://github.com/Esri/calcite-design-system/assets/821864/a1a1ed95-5549-40ec-b097-16e1533017b0)